### DISCUSSION TITLE
Option to have VTX Admin save VTX settings to FC

### DISCUSSION THREAD

**Discussion by CMartin249** [ExpressLRS/ExpressLRS#1427] [Ideas]
VTX Admin works great to set the VTX configuration. It would be very beneficial to save the configuration for future flights. Currently, VTX config set by VTX Admin doesn't appear to be saved (CRAZYBEEF4SX1280 - SPI ELRS). When changing to new battery, VTX starts on the saved channel and doesn't switch until after RX/TX connects which can take several seconds sometimes longer. If another person is flying on the FC's saved channel, this screws up their flight. VTX Admin should save the configuration so video comes back faster and doesn't cause someone else to crash.

**Comment by AlessandroAU** [comment]
VTX admin does save settings to FC EEPROM. If it not working via SPI it's a BF issue.

**Reply by CMartin249** [reply]
Please see discussion here on potential fix for SPI not saving in EEPROM in BF causing LQ drops and disconnects.
https://github.com/betaflight/betaflight/pull/11573#issuecomment-1115248316

**Comment by CMartin249** [comment]
There may be a consistency issue. SPI is worse, but I’ve seen it not save using an EP2.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
